central/role/defaults/permission_sets.go

```diff
@@ -0,0 +1,172 @@
+package defaults
+
+import (
+	"github.com/stackrox/rox/central/role/resources"
+	rolePkg "github.com/stackrox/rox/central/role/validator"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/env"
+)
+
+// Postgres IDs for permission sets
+// The values are UUIDs taken in descending order from ffffffff-ffff-fff4-f5ff-ffffffffffff
+// Next ID: ffffffff-ffff-fff4-f5ff-fffffffffff4
+const (
+	adminPermissionSetID                 = "ffffffff-ffff-fff4-f5ff-ffffffffffff"
+	analystPermissionSetID               = "ffffffff-ffff-fff4-f5ff-fffffffffffe"
+	continuousIntegrationPermissionSetID = "ffffffff-ffff-fff4-f5ff-fffffffffffd"
+	nonePermissionSetID                  = "ffffffff-ffff-fff4-f5ff-fffffffffffc"
+	// TODO: ROX-14398 Remove ScopeManager default role
+	scopeManagerPermissionSetID       = "ffffffff-ffff-fff4-f5ff-fffffffffffb"
+	sensorCreatorPermissionSetID      = "ffffffff-ffff-fff4-f5ff-fffffffffffa"
+	vulnMgmtApproverPermissionSetID   = "ffffffff-ffff-fff4-f5ff-fffffffffff9"
+	vulnMgmtRequesterPermissionSetID  = "ffffffff-ffff-fff4-f5ff-fffffffffff8"
+	vulnReporterPermissionSetID       = "ffffffff-ffff-fff4-f5ff-fffffffffff7"
+	vulnMgmtPermissionSetID           = "ffffffff-ffff-fff4-f5ff-fffffffffff6"
+	networkGraphViewerPermissionSetID = "ffffffff-ffff-fff4-f5ff-fffffffffff5"
+)
+
+// Permission sets names for default roles are derived from role name. Additional permission sets for which ACS does
+// not ship built-in roles must be declared in following block.
+const (
+	// VulnerabilityManager permission set provides necessary privileges required to view and manage system vulnerabilities and its insights.
+	// This includes privileges to:
+	// - view cluster, node, namespace, deployments, images (along with its scan data), and vulnerability requests.
+	// - view and create requests to watch images for vulnerability insights.
+	// - view and request vulnerability deferrals or false positives. This does include permissions to approve vulnerability requests.
+	// - view and create vulnerability reports.
+	VulnerabilityManager = "Vulnerability Manager"
```

Comment:
Open to name suggestions but `Vulnerability Management` seems natural name for a permission set that gives access vulnerability management feature bundle.

I will be adding a separate perm set `Vulnerability Management Admin`.